Vagrantfile

```diff
@@ -9,7 +9,9 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
   config.vm.box = "ubuntu/trusty64"
-
+  config.vm.synced_folder "./", "/vagrant",
```

Comment:
@u932b There is some bug in windows which is causing this error. I am not sure how this solves my problem. I worked with professor and figured out that adding these lines in Vagrantfile would fix the bug. It did solve my problem.